tools/release-notes/release-notes.go

```diff
@@ -56,14 +57,14 @@ func (r *releaseData) writeHcoChanges(span string) error {
 	return nil
 }
 
-func (p *project) writeOtherChangesIfVersionUpdated(f *os.File) error {
-	span := fmt.Sprintf("%s..%s", p.previousTag, p.currentTag)
-	releaseNotes, err := p.gitGetReleaseNotes(span)
+func writeOtherChangesIfVersionUpdated(g *git.Project, f *os.File) error {
```

Comment:
Please consider to use `io.Writer` instead of `*os.File`

This will

Reply:
done